outputs/aws.go

```diff
@@ -135,6 +138,37 @@ func (c *Client) SendMessage(falcopayload types.FalcoPayload) {
 	c.PromStats.Outputs.With(map[string]string{"destination": "awssqs", "status": "ok"}).Inc()
 }
 
+// UploadS3 upload payload to S3
+func (c *Client) UploadS3(falcopayload types.FalcoPayload) {
+	f, _ := json.Marshal(falcopayload)
+
+	prefix := ""
+	if c.Config.AWS.S3.Prefix != "" {
+		prefix = c.Config.AWS.S3.Prefix
+	}
+
+	key := fmt.Sprintf("%s/payload-%s-%v.json", falcopayload.Rule, prefix, time.Now().UnixNano())
+	resp, err := s3.New(c.AWSSession).PutObject(&s3.PutObjectInput{
+		Bucket: aws.String(c.Config.AWS.S3.Bucket),
+		Key:    aws.String(key),
+		Body:   bytes.NewReader(f),
+	})
+	if err != nil {
+		go c.CountMetric("outputs", 1, []string{"output:awss3", "status:error"})
```

Comment:
you need to declare the stats in https://github.com/falcosecurity/falcosidekick/blob/master/stats.go